packages/expression-parser/src/expression-parser/BooleanExpressionParser.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { ExpressionParser } from './ExpressionParser';
+
+export class BooleanExpressionParser extends ExpressionParser {
+  constructor() {
+    super();
+    this.importFunctions();
+  }
```

Comment:
If this gets extended to any other `ExpressionParser`s, I'd suggest we have a stub `importFunctions` in the base class, so the children don't have to override the constructor